src/helper/build.ts

```diff
@@ -6,9 +6,32 @@ import webpack, { MultiStats } from 'webpack';
 import createBaseWebpackConfig from '../helper/theme.react.config';
 import fs from 'fs';
 import rimraf from 'rimraf';
+import Logger from '../lib/Logger';
 
 export const THEME_ENTRY_FILE = path.join('theme', 'index.js');
+export const VUE_THEME_ENTRY_FILE = path.join("..",'theme', 'index.js');
+export const DEV_VUE_THEME_ENTRY_FILE = path.join('theme', 'index.js');
 
+export const CDN_ENTRY_FILE =  path.join('.fdk', 'cdn_index.js');
+
+export const dynamicCDNScript = ({assetNormalizedBasePath,vueJs }) => {
+    return `function getCDNurl() {
+        \n let cdnUrl; 
+        \n try{ 
+            \n if(fynd_platform_cdn) { 
+                \n cdnUrl = fynd_platform_cdn 
+                \n } else { 
+                    \n throw new Error("undefiend variable")}  
```

Comment:
Why throwing error is needed when simply assignment of assetNormalizedBasePath is required? 
Also code needs to be formatted